flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -151,8 +151,6 @@
 
 	protected org.apache.flink.configuration.Configuration flinkConfiguration;
 
-	protected final boolean isNewMode = true;
```

Comment:
I would suggest to not remove this field until we have ported all test cases. Then, we would not have to add the `@Ignore` statements.

Reply:
OK, I will keep this field.